scheduler/server/job_state.go

```diff
@@ -170,12 +195,16 @@ func (j *jobState) taskCompleted(taskId string, running bool) {
 		delete(j.NotStarted, taskId)
 	}
 	j.Completed[taskId] = taskState
+
+	// remove the task from the map of tasks by start time
+	j.removeTaskFromStartTimeMap(taskState.JobId, taskId, startTimeSec)
```

Comment:
is this tasksByJobClassAndStartTimeSec just in progress tasks?

Reply:
yes